zenml/core/steps/trainer/pytorch_trainers/torch_base_trainer.py

```diff
@@ -0,0 +1,19 @@
+#  Copyright (c) maiot GmbH 2020. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from zenml.core.steps.trainer.base_trainer import BaseTrainerStep
```

Comment:
@bcdurak Perhaps this is a good place to actually check that pytorch is installed or not or that `pip install zenml[pytorch]` is installed specifically. Did you look at entrypoints for this?

Reply:
resolved it!